### ISSUE TITLE
[Minor Bug or a Feature Request] When Engine Hours is selected for a vehicle, the corresponding fields do not currently change

### ISSUE THREAD

**Issue by CivisMiles** [hargata/lubelog#484]
**Checklist**
Please make sure you have performed the following steps before opening a new bug ticket, change `[ ]` to `[x]` to mark it as done
- [x] I have read and tried the steps outlined in the [Troubleshooting Guide](https://docs.lubelogger.com/Troubleshooting)
- [x] I have searched through existing issues.

**Description**
When the Engine Hours is selected nothing seems to change and the corresponding fields do not change to reflect the option. The odometer tab stays named Odometer, when entering a reading it stats odometer reading, the reminders still lists odometer, and all the dropdown fields still give the typical mileage intervals instead of the typical hours intervals prefilled. The field should change to Hour Meter/Engine Hours/Hours and the Typical Engine hour intervals should be close to 10, 20, 50, 100, 150, 200, 300, 400, 500, 1000. The Screenshots are of the current behavior in the demo though it applies to the Docker image as well.

This very well might be feature request instead of a bug if this was overlooked. An actual feature request related to this is separate Reminder Urgency Thresholds for Engine Hours VS Mileage.

Thanks for all you do for this software.

**Platform**
Docker Image
Online Demo

**Screenshots(optional)**
![image](https://github.com/hargata/lubelog/assets/7795937/6b958c0a-27ec-44fb-9f80-218d42f2b01d)
![image](https://github.com/hargata/lubelog/assets/7795937/6520c9c3-9406-46e1-adf6-be69676a2bc0)
![image](https://github.com/hargata/lubelog/assets/7795937/2b5e66c7-5063-4553-be29-f91ae457578b)

**Comment by Timo993** [comment]
Thanks for reporting! I have the exact same problem.
Self-hosted Docker.

**Comment by rtrickett337** [comment]
Yes, same here. Would be great if the ‘Engine Hours’ toggle did something as currently I don’t see anything change. Would like to see this updated as many of my vehicles I track in hours.

**Comment by KK4BitsPerSecond** [comment]
I am also having this issue in my docker container.

Current workaround is to just log 1 hour as 1 mile, but its a bit tedious and it will mess with my calculations later when I try to look at fuel economy. I need to be able track distance traveled and engine hours separately as certain maintenance tasks are required at different intervals depending on the usage and the point of using the software is so I don't have to think about it.

Thanks for the hard work.

**Comment by neptuneh123** [comment]
I'll add that I'm also seeing this in 1.5.3 (docker-self hosted) as well.

Thanks so much to the devs for their work on this! This is a great project and I'm really enjoying the features that have been added!

**Comment by hargata** [comment]
Next release:

Miles:
<img width="181" height="349" alt="Image" src="https://github.com/user-attachments/assets/4c20dd1f-5447-43d2-9be3-d4806badb33a" />

Hours:
<img width="152" height="310" alt="Image" src="https://github.com/user-attachments/assets/20ee6427-faba-401a-9cda-5e720c719407" />

Kilometers:
<img width="222" height="321" alt="Image" src="https://github.com/user-attachments/assets/99930d37-6d3b-4a69-994b-73b2f953e13e" />

Supports translations as well:

<img width="167" height="303" alt="Image" src="https://github.com/user-attachments/assets/08a222c6-07e7-435a-97cd-54d9760c4f23" />

Regarding the intervals that are more suited for engine hours, you should use the "Other" option instead to set custom intervals.
